tests/data/synthetic pre2018 Single cycle relative times.txt

```diff
@@ -0,0 +1,8 @@
+400.0	12345.000
```

Comment:
Hm, does this work even without the header line? Interesting!

Reply:
Yes, none of the `pre2018` data that I have seen have headers here. The column designation is inferred from the order and added in the code.